tests/testthat/test-betaDiv.R

```diff
@@ -39,7 +40,7 @@ test_that('betaDiv returns a correctly formatted data.table', {
               recordIdColumn = 'entity.SampleID')
 
   dt <- betaDiv(data, method='bray', verbose=F)@data
-  expect_equal(nrow(dt), nrow(df))
+  expect_equal(nrow(dt), nrow(df) - nNAs)
```

Comment:
if we don't remove empty samples, shouldn't this be just nrow(df)? The nNAs should still be in there